database/migrations/2020_07_07_193839_create_companies_table.php

```diff
@@ -26,6 +30,6 @@ public function up()
      */
     public function down()
     {
-        Schema::dropIfExists('m_d_a_s');
+        Schema::dropIfExists('companies');
```

Comment:
here

Reply:
@AironDev Well, two people can bear the same names. I don't think there is a need for the names to be unique.

Reply:
yeah @papa28x4, company name (companies) has to be unique because of foreign key constraints in payments table. maybe instead of creating FK constraint, we can simply add company_id to payments then define the relationship in the models